### PR TITLE
Paketo-ize

### PR DESCRIPTION
**Pull request by nebhale**
This change updates the command-function-buildpack to be in the style of the new Paketo buildpacks.  It now uses the `libcnb` and `libpak` libraries and adopts the declarative build plan style of cross-buildpack coordination.

### CODE REVIEW

File: buildpack.toml [hunk 1]
```diff
@@ -15,39 +15,48 @@
...
 id = "io.buildpacks.stacks.bionic"
 
 [[stacks]]
 id = "org.cloudfoundry.stacks.cflinuxfs3"
 
-[[stacks]]
-id = "heroku-18"
+[[metadata.configurations]]
+name        = "RIFF"
+description = "whether this is a riff application without a riff.toml file"
```
Comment (scothis) [inline buildpack.toml]:
```suggestion
description = "whether this is a riff function without a riff.toml file"
```

File: buildpack.toml [hunk 2]
```diff
@@ -15,39 +15,48 @@
 api = "0.2"
 
 [buildpack]
-id = "io.projectriff.command"
-name = "Command Function Buildpack"
-version = "0.2.0-BUILD-SNAPSHOT"
+id       = "riff-buildpacks/command-function"
```
Comment (scothis) [inline buildpack.toml]: not sure about this suggestion, take it or leave it

```suggestion
id       = "projectriff/command-function"
```

**Review by scothis — COMMENTED** [review]
Previously we had acceptance tests that would create a builder with just this buildpack and then build/invoke test functions. How is acceptance testing handled in this new world?

**Comment by nebhale** [comment]
We do the testing downstream in the builder since so many of the buildpacks can no longer operate in isolation from one another.  For example, both the `java-` and `node-` buildpacks now require `streaming-http-adapter-` to properly trigger.

**Review by scothis — APPROVED** [review]

**Pull request merged**